arrow/formatter.py

```diff
@@ -8,6 +8,14 @@
 
 from arrow import locales, util
 
+FORMAT_COOKIE = "dddd, DD-MMM-YYYY HH:mm:ss ZZZ"
+FORMAT_RFC822 = "ddd, DD MMM YY HH:mm:ss Z"
+FORMAT_RFC850 = "dddd, DD-MMM-YY HH:mm:ss ZZZ"
+FORMAT_RFC1036 = "ddd, DD MMM YY HH:mm:ss Z"
+FORMAT_RFC1123 = "ddd, DD MMM YYYY HH:mm:ss Z"
+FORMAT_RFC2822 = "ddd, DD MMM YYYY HH:mm:ss Z"
+FORMAT_RSS = "ddd, DD MMM YYYY HH:mm:ss Z"
```

Comment:
Seems like we are missing a few formats from here: https://pendulum.eustace.io/docs/#common-formats.

Formats like `atom`, `rfc3339`, and `w3c`. Is RFC 3339 the same as ISO 8601? Or should we put a format string for both?

Reply:
From what I can see `atom`, `rfc3339` and `w3c` are all the same and identical to ISO 8601.

Also  `RFC1123` and `RFC2822` are identical, do you think we need both?

Reply:
I am wondering if it is worth it to make "aliases" for each one of these, just so someone can easily reference them. Because someone who wants the atom format may not know it is rfc3339, for example.

What do you think?

Reply:
I like the idea, how would we implement the aliases?

Reply:
Could maybe do this: `FORMAT_RSS = FORMAT_RC1123 = FORMAT_RFC2822 = "ddd, DD MMM YYYY HH:mm:ss Z"`

Thoughts?

Reply:
Sounds good I'll make the change.

Reply:
Hey Jad I had a look at implementing this change, and from my reading I think we should avoid it and accept some duplication in favour of usability and simplicity.

Reply:
Yeah, they will all reference the same string (if you run `id()` on the variables, they will all be the same).

Don't you think we should include an obvious format like iso8601, even if it is a duplicate?